server/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - PyPika: Athena translator now uses `DATE_PARSE` rather than `TIMESTAMP` for the `todate` step
 - PyPika: Postgres and Redshift translator now use `TO_DATE` rather than `TIMESTAMP` for the `todate` step
+- PyPika: The `text` step now does an explicit cast of the input text to the adequate text type
```

Comment:
```suggestion
- PyPika: The `text` step now does an explicit cast of the input text to the corresponding 'DATA_TYPE_MAPPING' text type
```

?

Reply:
Users do not care about how our variables are named